### PR TITLE
HousingCompany and Apartment models and serialization

### PR DESCRIPTION
**Pull request by ehaivala**
Implement HousingCompany and Apartment XMLModels and every model related to them. Add factories for every model. Implement serialization test for all HousingCompany models.

At this stage test only that the Apartment serialization does not crash. More thorough tests should be done in separate PR. Leave them out for now to keep things moving forward.

Refs ASU-388

### CODE REVIEW

File: django_oikotie/enums.py [hunk 1]
```diff
@@ -12,32 +12,37 @@ class ApartmentAction(Enum):
     REMOVE = "remove"
 
 
+class Availability(Enum):
+    pre_marketing = "ennakkomarkkinoinnissa"
```
Comment (klemmari1) [inline django_oikotie/enums.py:16]: Are these left lower case intentionally?.

Comment (ehaivala) [reply django_oikotie/enums.py:16]: Ah, no they're not. I'll fix those.

Comment (ehaivala) [reply django_oikotie/enums.py]: And fixed. Found other enums with lowercase names as well.

**Review by klemmari1 — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**